multiqc/utils/config_defaults.yaml

```
@@ -271,10 +274,11 @@ fn_ignore_files:
   - "*.gtf"
   - "*.bed"
   - "*.vcf"
+  - "*.tbi"
   - "*.txt.gz"
   - "*.pdf"
-  - "*[!_][!m][!q][!c].html"
   - "*.md5"
+  - "*[!_\\.][!mv][!qe][!cp].html"
```

Adding on something I was thinking about the other day whilst we're here..

```suggestion
  - "*[!_\\.][!mv][!qe][!cp].html" # Allow _mqc.html and _vep.html files
  - "*.py[cod]" # Compiled python files, eg. MultiQC installation directory
```